src/services/ethereum/processor.ts

```diff
@@ -121,7 +132,7 @@ export class Processor {
       tgBot.executed({ value: assetValue, savings: gasSaved });
 
       webhookService.notify({ ...scheduled.toJSON(), status, gasPaid, gasSaved } as IScheduled);
-
+      await this.markLowerPriorityTransactions(scheduled, transactonList);
```

Comment:
rename to be more descriptive - mark as what?

Reply:
If we got to this point then there's no need to process the other txs in the group - we should break the loop `processTransaction` (line 63). You may need to return some extra parameter to tell `processTransaction` to break